### PR TITLE
Fix missing images from release tarball

### PR DESCRIPTION
**Pull request by lmm**
## Description

This fixes: https://github.com/projectcalico/calico/issues/3075

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
Fix missing images from release artifact
```

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit 4a2b9f45e6cbc2928678738fd5df21ee2d7619f7

https://deploy-preview-3080--calico.netlify.com

File: Makefile [hunk 1]
```diff
@@ -40,6 +40,8 @@ NODE_VER := $(shell cat $(VERSIONS_FILE) | $(YAML_CMD) read - '"$(RELEASE_STREAM
 CTL_VER := $(shell cat $(VERSIONS_FILE) | $(YAML_CMD) read - '"$(RELEASE_STREAM)".[0].components.calicoctl.version')
 CNI_VER := $(shell cat $(VERSIONS_FILE) | $(YAML_CMD) read - '"$(RELEASE_STREAM)".[0].components.calico/cni.version')
 KUBE_CONTROLLERS_VER := $(shell cat $(VERSIONS_FILE) | $(YAML_CMD) read - '"$(RELEASE_STREAM)".[0].components.calico/kube-controllers.version')
+POD2DAEMON_VER := $(shell cat $(VERSIONS_FILE) | $(YAML_CMD) read - '"$(RELEASE_STREAM)".[0].components.flexvol.version')
```
Comment (caseydavenport) [inline Makefile]: Do we need the flannel migration controller here as well?

Comment (lmm) [reply Makefile]: I had added that but removed it. My thinking was that it wasn't really a mainline installation (e.g. we don't include `flannel` for canal installations either. But I can add it in.

Comment (lmm) [reply Makefile]: I've added flannel-migration-controller 0fc51c12cdff32222c56b27e2307dccc1b85769e

**Pull request merged**